### PR TITLE
Explain why the Bevy editor can't just be a plugin for existing tools

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Prompted by discussion on [Discord](https://discord.com/channels/691052431525675048/789987800014651412/1290697160907817072).

### CODE REVIEW

**Comment by AmitHossain** [comment]
It is very rare that games of any significant scope are made without designers, artists, animators, and/or other asset creators. An editor is a core part of game engines. By including an editor by default, it would empower asset creators to use the engine, rather than just rely on the programmers. It would also make Bevy less intimidating for non-programmers, allowing for the growth and proliferation of the Bevy engine amongst game devs, and thus lead to more contributors helping build the engine, both in terms of numbers and diversity of backgrounds/experiences.

**Comment by alice-i-cecile** [comment]
> It is very rare that games of any significant scope are made without designers, artists, animators, and/or other asset creators. An editor is a core part of game engines. By including an editor by default, it would empower asset creators to use the engine, rather than just rely on the programmers. It would also make Bevy less intimidating for non-programmers, allowing for the growth and proliferation of the Bevy engine amongst game devs, and thus lead to more contributors helping build the engine, both in terms of numbers and diversity of backgrounds/experiences.

Ah, as an argument for why this shouldn't just be a plugin? I can get behind that. Let me try and make this more succinct.

**Comment by CalculatedToast** [comment]
> > It is very rare that games of any significant scope are made without designers, artists, animators, and/or other asset creators. An editor is a core part of game engines. By including an editor by default, it would empower asset creators to use the engine, rather than just rely on the programmers. It would also make Bevy less intimidating for non-programmers, allowing for the growth and proliferation of the Bevy engine amongst game devs, and thus lead to more contributors helping build the engine, both in terms of numbers and diversity of backgrounds/experiences.
> 
> Ah, as an argument for why this shouldn't just be a plugin? I can get behind that. Let me try and make this more succinct.

I can be a bit verbose. 😅 

If not technically, then at least philosophically, a plugin implies that something is optional and not a core part of the engine (at least from my non-programmer background). I believe that an editor _should_ be core to the engine, as pretty much all non-programmer game devs cannot work without it.

**Review by Atlas16A — APPROVED** [review]

File: design-book/src/design-constraints.md [hunk 1]
```diff
@@ -53,6 +54,14 @@ Below each bullet point, motivation is given.
   - Dogfooding and improving `bevy_ui` is a major goal. If something sucks, fix it upstream!
 - No scripting languages.
   - This targets a very niche audience: Rust is safer and easier to write than C++ and pure artists won't write Lua either. Prefer GUI controls, and then defer to node-based logic for highly complex art/design workflows.
+- The Bevy Editor is not a plugin for a third-party editor like Blender or LDTK. While tools like [Blenvy](https://github.com/kaosat-dev/Blenvy) can be very productive for a team making a specific game, they're not a replacement for the Bevy Editor.
+  - The existence of a standalone Bevy Editor tool (even with relatively limited capabilities) is valuable for helping new users get comfortable with the idea of making games in Bevy and become a functional member of a team.
```
Comment (n1ght-hunter) [inline design-book/src/design-constraints.md:58]: i would possibly say here that the editor will be extendable with plugins same as bevy. to off shoot limited capabilities

**Comment by MiniaczQ** [comment]
Using "plugin" here is a bit confusing, for a second I thought bevy-editor would require an alternate method of connecting to bevy apps that isn't a plugin

**Comment by Cedar-81** [comment]
Something to consider is the Next Generation Scene Ui proposition that is in the works, to take full advantage of that and show its capabilities bevy needs its own standalone editor that is fully customized for what innovations bevy decides to bring about in the future. Using a different existing tool will lead to a lot of scrambling to get things to work

File: design-book/src/design-constraints.md (hunk 1 above)
Comment (alice-i-cecile) [reply design-book/src/design-constraints.md:58]: Yeah, that's listed in the roadmap under uncategorized work :)

**Pull request merged**